modules/scalacheck/test/src/weaver/scalacheck/PropertyDogFoodTest.scala

```diff
@@ -31,7 +34,7 @@ object PropertyDogFoodTest extends IOSuite {
   // 100 checks sleeping 1 second each should not take 100 seconds
   test("Checks are parallelised") { dogfood =>
     for {
-      (_, events) <- dogfood.runSuite(Meta.ParallelChecks)
+      events <- dogfood.runSuite(Meta.ParallelChecks).map(_._2)
```

Comment:
IIRC that's because better-monadic-for is not available on Scala 3.

Reply:
That makes me SO GODDAMN ANGRY